### PR TITLE
add header && icons

### PR DESCRIPTION
**Pull request by pakoval**

### CODE REVIEW

File: src/components/Header.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <ul class="header__nav-list">
+        <li
+          v-for="item in navList"
+          :key="item.title"
+          class="header__nav-item"
+        >
+          <router-link :to="item.path" class="header__nav-link">
+            <IconSvg
+              v-if="item.icon"
+              :icon-name="item.icon.name"
```
Comment (silchencko) [inline src/components/Header.vue]: I'd make a single property "icon" and pass all icon parameters as a single object

File: src/components/IconSvg.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  >
+    <use :xlink:href="`/sprite.svg#${iconName}`"></use>
+  </svg>
+</template>
+
+<script>
+export default {
+  name: "IconSvg",
+  props: {
+    iconName: {
```
Comment (silchencko) [inline src/components/IconSvg.vue]: You can define a single prop, i wrote about before, here

**Review by silchencko — COMMENTED** [review]
It really looks good. I wish I could run it locally to test better, but I unfortunatelly I can't do it now. Could you add screenshots in the pr description and share your thoughts about props (see comments)?

File: src/components/Header.vue (hunk 1 above)
Comment (pakoval) [reply src/components/Header.vue]: <IconSvg
              v-if="item.icon"
              :icon="item.icon"
            />
            
            Do u mean like this?

**Review by silchencko — APPROVED** [review]
Good job!

**Pull request merged**